Adafruit_SH1106.cpp

```diff
@@ -26,12 +26,16 @@ However, SH1106 driver don't provide several functions such as scroll commands.
 
 *********************************************************************/
 
-#include <avr/pgmspace.h>
-#ifndef __SAM3X8E__
- #include <util/delay.h>
+#ifdef __AVR__
+    #include <avr/pgmspace.h>
+    #ifndef __SAM3X8E__
```

Comment:
This conditional is now superfluous because `__SAM3X8E__` will never be defined when `__AVR__` is defined (because the "SAM3X8E" microcontroller is not of the AVR architecture).